collect_app/src/test/java/org/odk/collect/android/utilities/FormNameUtilsTest.java

```diff
@@ -20,9 +20,9 @@
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNull;
-import static org.odk.collect.android.utilities.RegexUtils.normalizeFormName;
+import static org.odk.collect.android.utilities.FormNameUtils.normalizeFormName;
```

Comment:
You might also consider a static import for `formatFilenameFromFormName` to cut down the noise.